internal/pkg/model/component.go

```diff
@@ -155,4 +159,27 @@ func (c *ComponentsMap) doSet(component *Component) {
 	c.mutex.Lock()
 	defer c.mutex.Unlock()
 	c.components[component.String()] = component
+	if component.Data.DefaultBucket && component.Data.DefaultBucketStage != "" {
+		c.addDefaultBucket(component)
+	}
+}
+
+func (c *ComponentsMap) addDefaultBucket(component *Component) {
+	r := regexp.MustCompile(`(?i)[^a-zA-Z0-9-]`)
```

Comment:
Nejsem si jistej jak tohle funguje a jestli je ok, že se ten regex "kompiluje" při každé iteraci a jestli by se to nemělo provést jen jednou.

Reply:
mozes si ho ulozit v `NewComponentsMap`.
alebo namiesto balicka `regexp.` pouzit `regexpcache.` (rovnaky interface), uz sa niekde v projekte pouziva.